tests/client/coll.rs

```diff
@@ -0,0 +1,50 @@
+use bson::Document;
+use bson::Bson;
+use mongodb::client::wire_protocol::operations::{OpQueryFlags, Message};
+use mongodb::client::MongoClient;
+use mongodb::client::db::Database;
+use mongodb::client::coll::{Collection, FindOptions};
+use std::io::Write;
+use std::net::TcpStream;
+
+//#[test]
+fn find() {
+    let client = MongoClient::with_uri("mongodb://localhost:27017").unwrap();
+    let db = client.db("sample");
+    let coll = db.collection("movies");
+    
+    let results = coll.find(None, None).unwrap();
+
+    assert!(results.len() > 0);
+    let expected_val = &Bson::String("Jaws".to_owned());
```

Comment:
These tests (including the current ```wire_protocol``` tests) are all defunct since they rely on a custom local collection. Until inserts are supported, there isn't much use in having these tests in the codebase beyond being an example of library usage. Should I remove these for now?